music_assistant/providers/opensubsonic/sonic_provider.py

```diff
@@ -776,6 +776,72 @@ def _streamer() -> None:
 
         self.logger.debug("Done streaming %s", streamdetails.item_id)
 
+    async def _get_podcast_channel_async(self, chan_id: str) -> None:
+        chan = await self._run_async(self.conn.get_podcasts, inc_episodes=True, pid=chan_id)
+        await self.mass.cache.set(key=chan_id, data=chan, expiration=3600)
```

Comment:
the channel id is maybe a bit too generic as cache key.
I would recommend also defining a cache category in the mass global constants (around line 700) and assign that category to all of your cache set/get actions. It is also faster because it's indexed so the cache lookup can first do an indexed lookup on the category aimed at subsonic and only has to do the text search